src/pages/minjungkim/Login/components/LoginForm.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+
+class LoginForm extends Component {
+  state = {
+    loginId: '',
+    loginPassword: '',
+  };
+
+  handleValueChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
```

Comment:
구조분해할당을 사용해도 좋을 것 같아요 :)
```suggestion
  handleValueChange = e => {
     const { name, value } = e.target;
    this.setState({ [name]: value });
  };
```

Reply:
유림님꺼 보고 저도 아차 싶었어요 :) 땡큐!